app/Transformers/UserStatisticsTransformer.php

```diff
@@ -28,6 +28,7 @@ public function transform(UserStatistics\Model $stats = null)
                 'progress' => $stats->currentLevelProgressPercent(),
             ],
 
+            'global_rank' => $stats->globalRank(),
```

Comment:
Apparently I used `global_rank` for variants (in this file) and `rank_global` for scores 🤨